Sources/ReleaseServer/ReleaseServer.WebApi/Extensions/ByteArrayValidationExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ReleaseServer.WebApi.Extensions
+{
+    public static class ByteArrayValidationExtensions
+    {
+        public static bool IsAValidJson(this byte[] byteArray, out string exceptionMessage)
```

Comment:
Using a method like that is not good practice as you are deserializing an object that will be deserialized anyway in a later step if it is valid JSON.
So in the case of valid JSON the deserialization will be done twice.

When you are deserializing the object to e.g. `DeploymentInfoModel` it will also be checked if the the JSON is valid so there is no need for a check beforehand.

Reply:
I eliminated this function, because it's not needed anymore.

Reply:
See: https://github.com/Traeger-GmbH/release-server/commit/3f9d9fb070b98c88cbe9b747abc28b7f2aee3e72